RubeusCore/Source/logger_component.cpp

```diff
@@ -31,3 +31,28 @@ void DevILClearError()
 	while (ilGetError() != IL_NO_ERROR)
 		;
 }
+std::map<std::string, short> ColorCoder::foregroundColorMap={
```

Comment:
The problem here is that you are calling the map and the data inside it before the data has been input into the map. This creates a situation of you trying to access a part of the memory that you cannot. So shift these definitions somewhere so that there is an absolute guarantee that the map that you are accessing is indeed containing the data that you need.

Reply:
i was following this-
https://stackoverflow.com/questions/2616906/how-do-i-output-coloured-text-to-a-linux-terminal